Wiley Online Library.js

```diff
@@ -85,8 +85,8 @@ function processSubtitles(doc, item) {
 function addBookReviewTag(doc, item) {
 	var primaryHeading = ZU.xpathText(doc, '//span[@class="primary-heading"]');
 	if (primaryHeading) {
-		primaryHeading = primaryHeading.trim().replace(/book\s?review.*/i, 'Book Review');
-		if (primaryHeading.match(/(Book Review)|(Review Essays?)|(reviews?)/))
+		primaryHeading = primaryHeading.trim().replace(/(book\s?review.*)|(reviews?.*)|(review\sessays?.*)/i, 'Book Review');
```

Comment:
Looks fairly complicated: Why isn't 
 ```js
if (primaryHeading.match(/Book Review|Review Essays?|Reviews?/i))
      item.tags.push('Book Review');
```
sufficient?

Reply:
No. If the output is e.g. "review, review", i want to clean unwanted double tag

Reply:
But in effect you replace the whole expression by "Book Review" then test the match again in  line 89f and add to tags afterwards. The original value of `primary heading` is not kept.

Reply:
Oh, yes. Line 88 should be after line 89.

Reply:
But still with the current version, all "review-related" things are replaced by 'Book Review', all other results are inserted to tags unconditionally, although they are possibly entirely unrelated and the function is called `addBookReviewTag()`

Reply:
review-related tags should be intentionally replaced by "Book Review" to create "Rezension" as GND.

Reply:
Yes, of course. So why not just stick to that in this function, i.e. generate a "Book Review" tag if appropriate and handle possible other keywords - if they can arise at all - at some other place.

Reply:
primaryHeading is just a category, not keywords. so you don't have to check additionally if there are other keywords.